alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
```

Comment:
It would be cool to run the integration tests against nginx later, and not a local server with different performance (especially concurrency!) behavior on asset delivery.

But for now the anwer is: No we do not need nginx.

We should have runit set-up and ruby in the base container.